src/librbd/cache/pwl/Types.h

```diff
@@ -4,10 +4,20 @@
 #ifndef CEPH_LIBRBD_CACHE_RWL_TYPES_H
 #define CEPH_LIBRBD_CACHE_RWL_TYPES_H
 
+#include "acconfig.h"
+
+#ifdef WITH_RBD_RWL
+#include "libpmemobj.h"
+#endif
+
 #include <vector>
-#include <libpmemobj.h>
 #include "librbd/BlockGuard.h"
 #include "librbd/io/Types.h"
+#include "common/Formatter.h"
```

Comment:
Nit: use the forward declaration a couple lines below

Reply:
Done